src/components/ec-smart-table/ec-smart-table.spec.js

```diff
@@ -547,6 +547,29 @@ describe('EcSmartTable', () => {
       }]);
     });
 
+    it('should trigger fetch when filter prop is changed', async () => {
```

Comment:
Normally it shouldn't as in this case we can't say whether this event was triggered by changes from props or by changes which were done by user. But

Reply:
I think in both cases it's necessary to fetch benes because filters were changed and also this test is just to check if the model is being updated (checking the watch we added) to avoid having the same issue in the future

Reply:
To check whether model is updated we should check filters snapshot before the change and after. This event triggering test could stay as well.

Reply:
okay! I'm going to add a snapshot test as well